package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "selenium-standalone",
-  "version": "9.0.4",
+  "version": "9.0.5",
```

Comment:
This happens automatically during our release process

Reply:
will back it, thanks

Reply:
@christian-bromann done